Analyzers.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
Maybe rename to CodeAnalysis.props?

Reply:
the name is from <https://github.com/dotnet/runtime/blob/master/eng/Analyzers.props> but I can change if you would like?

Reply:
My thought is that the name like `CodeAnalysis.ruleset`.

Reply:
My preference is `Analyzers.props` for consistency with both dotnet/runtime repo and the suffix from analyzer package names, i.e. `microsoft.codeanalysis.analyzers`, `stylecop.analyzers`, `xunit.analyzers`